### PR TITLE
Fix sent_at typo in fillable array

### PR DESCRIPTION
**Pull request by azarzag**

### CODE REVIEW

**Comment by atymic** [comment]
@thomasjohnkane looks like travis is borked. I can't fix it as I don't have privs.

**Comment by azarzag** [comment]
@atymic and @thomasjohnkane any idea when this PR will be merged? 

Notification rescheduling is completely broken since the sent_at value is never reset.

**Comment by atymic** [comment]
@thomasjohnkane can you fix Travis? I can't merge this without permissions (as the checks are not passing).

File: src/Models/ScheduledNotification.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ class ScheduledNotification extends Model
         'notification_type',
         'notification',
         'send_at',
-        'sent',
+        'sent_at',
```
Comment (atymic) [inline src/Models/ScheduledNotification.php:34]: while this should be corrected, we don't used mass assignment when rescheduling:
https://github.com/thomasjohnkane/snooze/blob/master/src/Models/ScheduledNotification.php#L147

So the cause of the bug must be elsewhere. Could you open an issue with more details?

**Comment by atymic** [comment]
cc @thomasjohnkane

**Review by atymic — APPROVED** [review]

**Pull request merged**